src/anchor.js

```diff
@@ -1,5 +1,8 @@
 "use strict";
-
+/**
+ * @typedef Document
```

Comment:
Can you change these to use the standard one line syntax?

Reply:
sure